help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -0,0 +1,218 @@
+
+#### You need to have a Microsoft Dynamics 365 account {#prerequisites-account}
+
+Go to the [!DNL Dynamics 365] [trial](https://dynamics.microsoft.com/en-us/dynamics-365-free-trial/) page to register and create an account, if you do not have one already.
+
+#### Create field within Dynamics 365 {#prerequisites-custom-field}
+
+Create the custom field of type `Simple` with field data type as `Single Line of Text` which Experience Platform will use to update the segment status within [!DNL Dynamics 365].
+Refer to the [!DNL Dynamics 365] documentation to [create a field (attribute)](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1) if you need additional guidance.
+
+An example setup within [!DNL Dynamics 365] is shown below:
+![Target mapping](../../assets/catalog/crm/microsoft-dynamics-365/dynamics-365-fields.png)
+
+#### Gather Dynamics 365 credentials {#gather-credentials}
+
+Note down the items below before you authenticate to the [!DNL Microsoft Dynamics 365] CRM destination:
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| Username | Your [!DNL Dynamics 365] account username | MyDynamicsUsername |
+| Password | Your [!DNL Dynamics 365] account password | MyDynamicsPassword |
+| Environment URL | See [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/org-service/discover-url-organization-organization-service?view=op-9-1) for additional guidance. | If your domain is as below, you need the highlighted value.<br> *`org57771b33`.crm.dynamics.com* |
+
+## Supported identities {#supported-identities}
+
+[!DNL Microsoft Dynamics 365] supports update of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Example|Description|Considerations|
+|---|---|---|---|
+| `contactId` | `7eb682f1-ca75-e511-80d4-00155d2a68d1` | Unique identifier for a contact.| **Mandatory**, Refer to the [[!DNL Microsoft Dynamics 365] documentation](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/entities/contact?view=op-9-1) for further details. |
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | <ul><li>You are exporting all members of a segment, together with the desired schema fields *(for example: email address, phone number, last name)*, according to your field mapping.</li><li> Platform segment statuses are exported to [!DNL Dynamics 365] by specifying their corresponding custom field attribute in [!DNL Dynamics 365], within the **[!UICONTROL Activate Destination]** > **[!UICONTROL Schedule segment export]** > **[!UICONTROL Mapping ID]** field.</li></ul> |
```

Comment:
On line 69 we need to rephrase the second bullet point:

- Platform segment statuses are exported to [!DNL Dynamics 365] by entering their [!DNL Dynamics 365]  custom field attribute in the [!UICONTROL Mapping ID] field, during the [!UICONTROL Schedule segment export] step.

Reply:
There is a section #schedule-segment-export-example should I also link to it ?

Reply:
@vladfalon responded to similar query within https://github.com/AdobeDocs/experience-platform.en/pull/93#discussion_r943427991